src/util/kubeUtils.ts

```diff
@@ -0,0 +1,38 @@
+import { QuickPickItem } from "vscode";
+import { KubeConfig } from '@kubernetes/client-node';
+
+export async function getClusters(): Promise<QuickPickItem[]> {
+    const kubectlConfig = new KubeConfig();
+    kubectlConfig.loadFromDefault();
+    if (!kubectlConfig) {
+        return [];
+    }
+    const currentContextServer = kubectlConfig.currentContext.split('/')[1];
+    const clusters = kubectlConfig.clusters || [];
+    return clusters.map((c) => {
+        return {
+            label: c.name,
+            description: c.name === currentContextServer ? 'Current Context' : ''
+        };
+    });
+}
+
+export async function getUsers(clusterUrl: string): Promise<QuickPickItem[]> {
+    const kubectlConfig = new KubeConfig();
+    kubectlConfig.loadFromDefault();
+    if (!kubectlConfig) {
+        return [];
+    }
+    const currentContextUser = kubectlConfig.currentContext.split('/')[2];
+    return kubectlConfig.users
```

Comment:
What if there is not .kube/config? You might need to use getUsers() method instead.